ldk/javascript/examples/self-test-loop/src/tests/screen/index.ts

```diff
@@ -36,6 +37,40 @@ export const testOCR = (): Promise<boolean> =>
     }
   });
 
+const writeWhisperFileEncoded = (label: string, body: string) =>
+  whisper.create({
+    label,
+    onClose: () => {
+      console.log(`Closed Whisper`);
+    },
+    components: [
+      { body, type: whisper.WhisperComponentType.Markdown },
+      {
+        body: '![image](https://raw.githubusercontent.com/open-olive/loop-development-kit/HELPS-3796-ocrFileEncoded/ldk/javascript/examples/self-test-loop/static/testocr.png)',
```

Comment:
for my education - is this really how we put images into these loops, or does this address get updated somehow to something more permanent?

Reply:
I was wondering the same thing, will this url still be valid after merging in/deleting branch `HELPS-3796-ocrFileEncoded`

Reply:
we could do something like this `https://github.com/open-olive/loop-development-kit/blob/${branch}/ldk/javascript/examples/self-test-loop/static/testocr.png`, where `branch` can be `develop`

Reply:
@racary-olive  You r right, this link includes the branch name within it. I'd like to try Jeff's suggestion